compose-files/secret_store.yml

```diff
@@ -44,7 +44,7 @@ services:
     command: nginx -g 'daemon off;'
 
   secret-store-signing-node:
-    image: parity/parity:stable
+    image: parity/parity:v2.3.2
```

Comment:
rather use `beta` than a static version number.

Reply:
`beta` is similarly broken

Reply:
I'm sorry. We messed up the docker images. It will be addressed the next days in a subsequent release.